jdi-light-examples/src/main/java/selenium/HomePage.java

```diff
@@ -1,4 +1,4 @@
-package selenium.seleniumPO;
+package selenium.selenium;
```

Comment:
Please change folder structure for this package here and below so that it would correspond to the style that was before

**After:**
![seleniumAfter](https://user-images.githubusercontent.com/49747933/70060426-95a3d600-15f3-11ea-9974-ef8ab4f53c4c.PNG) 
**Before:**
![seleniumBefore](https://user-images.githubusercontent.com/49747933/70060451-9f2d3e00-15f3-11ea-93a6-639257f5661f.PNG)